qemu/tests/blockdev_commit_filter_node_name.py

```diff
@@ -0,0 +1,48 @@
+from provider import job_utils
+from provider import backup_utils
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitFilter(BlockDevCommitTest):
+
+    def commit_snapshots(self):
+        device = self.params.get("device_tag")
+        device_params = self.params.object_params(device)
+        snapshot_tags = device_params["snapshot_tags"].split()
+        self.device_node = self.get_node_name(device)
+        options = ["speed", "filter-node-name"]
+        arguments = self.params.copy_from_keys(options)
+        arguments["speed"] = self.params["commit_speed"]
+        filter_node_name = self.params["filter-node-name"]
+        arguments["filter-node-name"] = filter_node_name
+        device = self.get_node_name(snapshot_tags[-1])
+        commit_cmd = backup_utils.block_commit_qmp_cmd
+        cmd, args = commit_cmd(device, **arguments)
+        self.main_vm.monitor.cmd(cmd, args)
+        job_id = args.get("job-id", device)
+        block_info = self.main_vm.monitor.info_block()
+        if filter_node_name not in block_info:
```

Comment:
it's better to make a specific check, i.e. when doing commit, the node-name should be the one set in filter-node-name.